xml/operations-general_troubleshooting.xml

```diff
@@ -9,6 +9,24 @@
   General troubleshooting procedures for resolving your cloud issues including
   steps for resolving service alarms and support contact information.
  </para>
+ <para>
+  Before contacting support to help you with a problem on &cloud;, it is
+  strongly recommended that you gather as much information as possible about
```

Comment:
we recommend gathering as much information...* (to use more active language)

Reply:
yes. I debate with myself about stuff such as this. Weird gerunds, infinitives, active/passive. There is awkward sounding language throughout the documentation. Some day when there's nothing else to do.